packages/react-kitchen-sink/tsdown.config.mjs

```
@@ -0,0 +1,9 @@
+import { defineConfig } from 'tsdown'
```

ESLint

[error] Unable to resolve path to module 'tsdown'.

(import-x/no-unresolved)